packages/extra/etcd/templates/hook/job.yaml

```
@@ -0,0 +1,84 @@
+        - name: helm
+          image: alpine/k8s:1.31.6
+          command:
+          - bash
+          args:
+            - -c
+            - |-
+              version=$(kubectl -n tenant-lllamnyp get configmap etcd-previous-version -ojsonpath='{.data.version}')
+              semver_less_than() {
+                local v1="$1"
+                local v2="$2"
+              
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)